### ISSUE TITLE
Enable Destination Acceptance Tests cross-language support

### ISSUE THREAD

**Issue by sherifnada** [airbytehq/airbyte#4698]

**Comment by ad-m** [comment]
@sherifnada Can you confirm that this issue is still relevant? I wonder how to handle the @marcosmarxm request for acceptance testing of destination connector in #7231.

**Comment by sherifnada** [comment]
@ad-m yup, still relevant. Currently the only available option is to write acceptance tests in java

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 365 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.
